packages/cli/src/api/extractors/index.ts

```diff
@@ -1,11 +1,11 @@
-import ora from "ora"
+import fs from "fs/promises";
 import babel from "./babel"
 
 const DEFAULT_EXTRACTORS: ExtractorType[] = [babel]
 
-export type BabelOptions = {
-  plugins?: Array<string>
-  presets?: Array<string>
+export type ParserOptions = {
+  decoratorsBeforeExport?: boolean
+  flow?: boolean,
```

Comment:
The general idea is to hide implementation details of extractor. User should not know that babel parser is used. Tomorrow we can switch parser to SWC for example without breaking anything